services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from db.models import Order, Ticket
+from django.contrib.auth import get_user_model
+
+
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str | None = None) -> Order:
+    with transaction.atomic():
```

Comment:
Use transaction.atomic as decorator instance of context manager. It makes your code more readable